lib/code_corps/policy/policy.ex

```diff
@@ -51,6 +51,9 @@ defmodule CodeCorps.Policy do
   defp can?(%User{} = current_user, :delete, %UserSkill{} = user_skill, %{}), do: Policy.UserSkill.delete?(current_user, user_skill)
   defp can?(%User{} = current_user, :create, %UserRole{} = user_role, %{}), do: Policy.UserRole.create?(current_user, user_role)
   defp can?(%User{} = current_user, :delete, %UserRole{} = user_role, %{}), do: Policy.UserRole.delete?(current_user, user_role)
+  defp can?(%User{} = user, :show, %StripeConnectAccount{} = stripe_connect_account, params), do: Policy.StripeConnectAccount.show?(user, params)
+  defp can?(%User{} = user, :create, %StripeConnectAccount{}, params), do: Policy.StripeConnectAccount.create?(user, params)
+  defp can?(%User{} = user, :update, %StripeConnectAccount{} = stripe_connect_account, params), do: Policy.StripeConnectAccount.update?(user, stripe_connect_account)
```

Comment:
`%{}` here, since we aren't using the params, but still want to typecheck.

You should be getting compiler warnings for these, so be sure to get those eliminated.